app/projects/forms.py

```diff
@@ -612,14 +637,19 @@ def clean_output_parameters_names(self):
 
 class COPCalculatorForm(OpenPlanModelForm):
     def __init__(self, *args, **kwargs):
-        # TODO these fields dont get translated currently
         super().__init__(*args, **kwargs)
         self.fields["temperature_high"] = DualNumberField(
             default=60, min=-273, param_name="temperature_high"
         )
         self.fields["temperature_low"] = DualNumberField(
             default=40, min=-273, param_name="temperature_low"
         )
+        # Reset labels, units, help text etc. (deleted when defining as DualNumberField)

```

Comment:
This is what happens then with the "Input timeseries" as well! I am certain of it